index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>My Portolio Project</title>
+</head>
+<body>
+
+    <header class ="page-header">
+
+        <div id="frame-container">
+            <h2 id="Welcome-logo">Welcome</h2>
+            <i class="fa fa-bars"></i>
+        </div>
+      
+    </header>
+    <div class="introduction-text">
+        <div>
+            <h2 id="Greeting-text">Hey there. I’m Shedrack!</h2>
+            <h2 id="profession-text"> I’m a software Engineer.</h2>
+        </div>
+
+
+    </div>
+    <div class="description">
+        <p>
+            I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see, which i assure you is high quality and you have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+    </div>
+    <div class="icon-bar">
+        
+
+        <i class="fa fa-twitter"></i>
+            <i class="fa fa-linkedin"></i>
+                 <i class="fa fa-medium"></i>
+                     <i class="fa fa-github"></i>
+                        <i class="fa fa-angellist"></i>
```

Comment:
Listing them will alter the figma design

Reply:
Hi, @Shedrack-Sunday I understand but I think it s better to use the ul tag for semantics reasons as I mentioned early. And do not worry about altering the design because  When you will use the List tag, you will just need to use CSS to style them and make them look like on Figma design. :+1: